src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -23,6 +23,8 @@
  * questions.
  */
 
+import module jdk.httpserver;
+
```

Comment:
The statement above seems to be causing a NullPointerException in the javadoc tool. I've filed a bug on it to find out what we need to do. But, it may mean reverting to "import com.sun.net.httpserver.*;"

Also, the two implNotes are combined in the output HTML. So I would re-order them so the existing one comes first.

In the meantime I will file a CSR with the additional doc text below